synapse/lib/lmdbslab.py

```diff
@@ -21,6 +22,52 @@ def __init__(self, db, dupsort):
 
 _DefaultDB = _LmdbDatabase(None, False)
 
+class SlabDict:
+
+    def __init__(self, slab, info, db=None, pref=b''):
+        self.db = db
+        self.slab = slab
+        self.info = info
+        self.pref = pref
+
+    def get(self, name, defval=None):
+        return self.info.get(name, defval)
```

Comment:
What if it is in the DB but not in the info cache?

Reply:
The use pattern for this object assumes that the info cache was lifted by the given prefix ( via GuidStor.props() or something like it ) before the object was constructed.  To make it more generically useful, i'll move the info/props lifting code into the SlabDict constructor.